service/src/test/java/bio/terra/profile/pact/provider/BPMProviderTest.java

```diff
@@ -29,20 +30,24 @@
 import org.junit.jupiter.api.TestTemplate;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.data.jdbc.JdbcRepositoriesAutoConfiguration;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.boot.test.web.server.LocalServerPort;
 import org.springframework.cache.CacheManager;
+import org.springframework.jdbc.core.ConnectionCallback;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
 
 @Tag("provider-test")
 @Provider("bpm-provider")
 @PactBroker
 // for local testing, put any test pacts in the service/pacts folder.
 // then comment out the above line, and uncomment the following line
-// @PactFolder("pacts")
+// @PactFolder("src/test/java/bio/terra/profile/service/pacts")
```

Comment:
The previous path was incorrect.

Reply:
I didn't really intend for it to be in the source code (since it isn't a java file).
The original path will work for something in the root directory of the service project.

My suggestion is to use the `src/test/resources` folder, as something that would both make sense in terms of normal project organizational conventions, and be more obvious as to the location being referenced.

Reply:
OK, when I loop back around to this file for the next set of endpoints I will move this elsewhere.